src/views/RemoteView.vue

```diff
@@ -70,12 +70,19 @@ export default {
       this.speed -= 5;
     },
     speedUp() {
-      if (this.speed === 60) return;
```

Comment:
Just a quick question, was this line meant to be remove ? I don't really know if some walkingpads can go to more than 6km/h